backend/utils/llm

```diff
@@ -0,0 +1,13 @@
+import os
+from dotenv import load_dotenv
+from mistralai.client import MistralClient
+from mistralai.models.chat_completion import ChatMessage
+
+load_dotenv()
+
+endpoint = os.environ["AZURE_AI_MISTRAL_LARGE_ENDPOINT"]
+api_key = os.environ["AZURE_AI_MISTRAL_LARGE_KEY"]
+model = "azureai"
+
+client = MistralClient(api_key=api_key,
+                       endpoint=endpoint)
```

Comment:
File is missing `.py` suffix

Reply:
I have updated this, although for some reason in GH it is now showing both files are there